backend/btrixcloud/orgs.py

```diff
@@ -330,17 +340,13 @@ async def get_max_concurrent_crawls(self, oid):
     async def add_crawl_files_to_org_bytes_stored(self, oid: uuid.UUID, size: int):
```

Comment:
Wonder if this function redundant with the more general 'inc_org_bytes_stored' now?

Reply:
Yup! I think it was a holdover from when we didn't have the ops methods available in the operator. Removed now.